APIv2 Docs (#4120)
Opentrons/opentrons: api/docs/source/v2/apiv2index.rst

    # metadata
    metadata = {
        'protocolName': 'My Protocol',

>         'author': 'your email address <your-email@address.com>',

Reviewer: This example was meant to be name and email, e.g. `Mike Cousins <mike@example.com>` like you might see in an email client. If that's too confusing, the example should be either name _or_ email address (without the arrow brackets)

   def run(protocol: protocol_api.ProtocolContext):
       pass

> The function must be named exactly ``run`` and must take exactly one mandatory argument (we recommend ``protocol`` since this argument represents the protocol that the robot will execute and is easy to parse if another editor needs to impliment or resuse your code).

Reviewer 2: ```suggestion
The function must be named exactly ``run`` and must take exactly one mandatory argument (we recommend ``protocol`` since this argument represents the protocol that the robot will execute and is easy to parse if another editor needs to implement or reuse your code).
```

The protocol context plays the same role as the ``robot``, ``labware``, ``instruments``, and ``modules`` objects in past versions of the API, with one important difference: it is only one object; and because it is passed in to your protocol rather than imported, it is possible for the API to be much more rigorous about separating simulation from reality.

 Since the *robot* now *runs the protocol*, rather than the *protocol running the robot*, the key point to note is that there is no longer any need to ``import opentrons`` at the top of every protocol.

Labware
^^^^^^^
Refer to `the python docs <https://docs.python.org/3/index.html>`_ for clarification on using variables and additional information about handling labware in the :ref:`new-labware` section.

> The next step is defining the labware required for your protocol. You tell the protocol context about labware by calling the method ``protocol.load_labware(name, slot)``. By implimenting this, the protocol receives context about what should be present on the deck, and where.

Reviewer 2: ```suggestion
The next step is defining the labware required for your protocol. You tell the protocol context about labware by calling the method ``protocol.load_labware(name, slot)``. By implementing this, the protocol receives context about what should be present on the deck, and where.
```

The name of a labware is a string that is different for each kind of labware. You can reference all available labware by using the Opentrons `Labware Library <https://labware.opentrons.com>`_.

> Note:the slot is the labelled location on the deck in which you've placed the labware. The available slots are numbered from 1-11.

Reviewer 2: ```suggestion
Note: the slot is the labelled location on the deck in which you've placed the labware. The available slots are numbered from 1-11.
```
